src/create-key.ts

```diff
@@ -0,0 +1,15 @@
+import { isObject } from './util';
+
+export const koSort = (opts?: any): object => {
```

Comment:
I totally agree.  ```deepSortObject```  it is.
Because we don't have the constraint of returning an object from ```deepSortObject``` as we had before I changed the implementation accordingly.
Basically I dropped ```fn(1) ->{opts:1}```  in favor of ```fn(1)->1```. The function is more readable and more accurate as we don't change the content of the options passed.